src/resources/breaking-changes.md

```diff
@@ -0,0 +1,821 @@
+---
+title: Breaking changes and deprecations
+description: A list of breaking changes by release in Dart.
+---
+
+This page lists all language and library breaking changes and deprecations in Dart,
+organized by release and area, to help Dart users understand and manage their
+impact. Complete release notes are available in the [Dart SDK changelog][changelog].
+The [breaking change policy][] document describes the policy and process
+around breaking changes and deprecations in Dart. 
+
+**This page includes two types of breaking changes, as well as deprecations**:
+* **Unversioned**: The compiler does not maintain backward compatibility, and
+  code may break as soon as you [upgrade your sdk version][sdk] if it relies on
+  the previous behavior.
+  
+  *These are the majority of changes and are not specially marked in this list.*
+* **Language versioned**: The compiler maintains backward compatibility for
+  existing code, and the behavior change only takes effect (potentially breaking
+  code that relies on the previous behavior) when you upgrade the
+  [language version] of your code.
+  
+  *These are marked "**Language versioned**"*.
+* **Deprecations**: The compiler maintains compatibility for deprecated code,
+  with a warning. Deprecations are then completely removed in the release
+  following their deprecation, breaking any code that relies on the previous behavior.
```

Comment:
In the past, deprecations have been removed a bit inconsistently. The next release, a few releases later, a major release (Dart 3). Is there a documented process we can refer to here? Or just mention a minimum instead?

Reply:
I can just remove the "in the release following their deprecation" and make it more general, thanks for letting me know!